exchange/exchange.go

```diff
@@ -175,6 +176,7 @@ func NewExchange(adapters map[openrtb_ext.BidderName]AdaptedBidder, cache prebid
 		requestSplitter:          requestSplitter,
 		macroReplacer:            macroReplacer,
 		floor:                    cfg.PriceFloors,
+		priceFloorFetcher:        priceFloorFetcher,
```

Comment:
We probably don't need to pass the entire `config.PriceFloors` object to the `exchange struct` given that we only check for the `Enabled` value from here on. Unless we plan to use other fields inside `floor` in the future, can we chage to a boolean?
```
82       macroReplacer            macros.Replacer
83 -     floor                    config.PriceFloors
   +     priceFloorEnabled        bool
84       priceFloorFetcher        floors.FloorFetcher
85   }
exchange/exchange.go
```

Reply:
Agree, we don't need entire object, it was passed if we need something from config in the future

For now i don't think we need entire object, i will make the change